<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeReportFinish</class>
 <widget class="QWidget" name="ChargeReportFinish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>45</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>35</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();
border-image: url(:/background1.png);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,11,3,1">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,9,6,3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="labelCANID">
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
border-image: \*url();
text-decoration: underline;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelEndTime_2">
              <property name="font">
               <font>
                <family>Sans Serif</family>
                <pointsize>20</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(255, 255, 255);
border-image: \*url();</string>
              </property>
              <property name="text">
               <string>号充电完成请拔枪并挂枪，可继续充电</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="TimeLimit" name="timeLimit" native="true"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>150</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>150</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-image: url(:/battery.png);</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="sizeConstraint">
                <enum>QLayout::SetMinAndMaxSize</enum>
               </property>
               <item>
                <widget class="QProgressBar" name="progressBar">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>84</width>
                   <height>125</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>84</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-image: \*url();
selection-background-color: rgb(0, 255, 0);
background-color: rgb(215, 248, 211);
font: 19pt &quot;Sans Serif&quot;;</string>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="textDirection">
                  <enum>QProgressBar::TopToBottom</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="16">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
              <property name="sizeConstraint">
               <enum>QLayout::SetMinAndMaxSize</enum>
              </property>
              <property name="horizontalSpacing">
               <number>12</number>
              </property>
              <item row="1" column="1">
               <widget class="QLabel" name="labelStartSOC">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="labelChargingTime">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelStartSOC1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>起始SOC:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="labelChargingEnergy1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>充电电量:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="labelChargingEnergy">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelStartTime1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>开始时间:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="labelStartTime">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelChargingTime1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>充电用时:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="labelSuspendReason1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>终止原因:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="labelSuspendReason">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>179</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="labelEndTime1">
                <property name="font">
                 <font>
                  <family>Sans Serif</family>
                  <pointsize>20</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
                </property>
                <property name="text">
                 <string>结束时间:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="labelEndTime">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Sunken</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6,0,1">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="buttonSave">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 27pt ; } </string>
              </property>
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>238</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonPrintTicket">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 27pt ; } </string>
              </property>
              <property name="text">
               <string>打印小票</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonStartCharging">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 27pt ; } </string>
              </property>
              <property name="text">
               <string>继续充电</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLimit</class>
   <extends>QWidget</extends>
   <header>TimeLimit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
